core/ui/src/components/settings/CloudLogManagerConfigureModal.vue

```diff
@@ -0,0 +1,326 @@
+            >
+            </NsTimePicker>
+          </cv-column>
+        </cv-row>
+      </cv-form>
+      <NsInlineNotification
+        v-if="!subscription"
+        kind="info"
+        :title="
+          $t('cloud_log_manager_forwarder.available_with_subscription_title')
+        "
+        :description="
+          $t('cloud_log_manager_forwarder.available_with_subscription')
+        "
+        :showCloseButton="false"
+        :actionLabel="$t('cloud_log_manager_forwarder.more_details')"
+        @action="goToMoreDetails"
+      />
+      <NsInlineNotification
+        v-if="error.setCloudLogManager"
+        kind="error"
+        :title="$t('system_logs.cannot_configure_forwarder')"
+        :description="error.setCloudLogManager"
+        :showCloseButton="false"
+      />
+    </template>
+    <template slot="secondary-button">{{ $t("common.cancel") }}</template>
+    <template slot="primary-button">{{ $t("common.configure") }}</template>
+  </NsModal>
+</template>
+
+<script>
+import to from "await-to-js";
+import { NsToggle, TaskService, DateTimeService } from "@nethserver/ns8-ui-lib";
+import { carbonPrefixMixin } from "@carbon/vue/src/mixins";
+
+export default {
+  name: "CloudLogManagerConfigureModal",
+  props: {
+    isShown: {
+      type: Boolean,
+      required: true,
+    },
+    lokiToEdit: {
+      type: Object,
+      required: true,
+    },
+    configuration: {
+      type: Object,
+      required: true,
+    },
+    subscription: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  components: {
+    NsToggle,
+  },
+  mixins: [carbonPrefixMixin, TaskService, DateTimeService],
+  data() {
+    return {
+      radioVal: "last_timestamp",
+      toggleEnabled: false,
+      address: "",
+      tenant: "",
+      date: "",
+      time: "",
+      last_timestamp_formatted: "",
+      loading: {
+        setCloudLogManager: false,
+      },
+      error: {
+        address: "",
+        tenant: "",
+        setCloudLogManager: "",
+      },
+      calOptions: {
+        dateFormat: "Y-m-d",
+      },
+    };
+  },
+  methods: {
+    onModalHidden() {
+      this.$emit("hide");
+    },
+    resetModal() {
+      this.radioVal = "last_timestamp";
+      this.toggleEnabled = false;
+      this.address = "";
+      this.tenant = "";
+      this.date = "";
+      this.time = "";
+      this.last_timestamp_formatted = "";
+    },
+    goToMoreDetails() {
+      window.open(
+        "https://www.nethesis.it/servizi/cloud-log-manager",
+        "_blank"
+      );
+    },
+    onModalShown() {
+      if (
+        this.configuration.status == "active" ||
+        this.configuration.status == "failed"
+      ) {
+        this.toggleEnabled = true;
+      }
+
+      if (this.configuration.address == "") {
+        this.address = "https://nar.nethesis.it";
+      } else {
+        this.address = this.configuration.address;
+      }
+
+      if (this.configuration.tenant == "") {
+        this.tenant = "clienteit";
```

Comment:
Just asking: is this the correct default value?

Reply:
No, this is only for Nethesis

Reply:
So I guess an empty string is better here